11_taxonomic_information.Rmd

```diff
@@ -164,7 +164,16 @@ assay(altExp(tse, "Family"), "counts")[1:5,1:7]
 ```
 
 `altExpNames` now consists of `Family` level data. This can be extended to use 
-any taxonomic level listed in `mia::taxonomyRanks(tse)`.   
+any taxonomic level listed in `mia::taxonomyRanks(tse)`.
+
+Rare taxa can also be aggregated into a single group "Other" instead of filtering them out. A suitable function for this is `agglomerateByPrevance`.
+
+```{r}
+altExp(tse, "Family_prevalence") <- agglomerateByPrevalence(tse, rank = "Family", other_label = "Other")
```

Comment:
Two points:

1. would be good to explicitly show how and which abundance and prevalence thresholds are used here to define/quantify "prevalence" i.e. just make those arguments explicit in the call 
2. to consider: this example uses altExp, which is a more complex structure within TreeSE; the example could be simplified and made more focused on the core topic of agglomeration by prevalence if we would not use the altExp here. But I have mixed feelings since in a way it is good to make users familiar with all these possibilities.. anyway I would consider removing the altExp part and just operating with the main tse directly(?). Family is also rather high level, usually the number of rare taxa is much higher on the ASV/species level and the need for agglomeration greater.

Reply:
2. Due to the other examples in the same chapter, I'd keep the altExp in this case as well